src/tokens/selectors.ts

```diff
@@ -357,7 +359,13 @@ export const swappableTokensByNetworkIdSelector = createSelector(
 export const cashInTokensByNetworkIdSelector = createSelector(
   (state: RootState, networkIds: NetworkId[]) => tokensListSelector(state, networkIds),
   (tokens) =>
-    tokens.filter((tokenInfo) => tokenInfo.isCashInEligible && isCicoToken(tokenInfo.symbol))
+    tokens.filter(
+      (tokenInfo) =>
+        tokenInfo.isCashInEligible &&
+        (getFeatureGate(StatsigFeatureGates.USE_CICO_CURRENCY_BOTTOM_SHEET) //TODO: Remove after CiCo currency bottom sheet is rolled out
+          ? isCicoToken(tokenInfo.symbol)
+          : true)
```

Comment:
```suggestion
        (getFeatureGate(StatsigFeatureGates.USE_CICO_CURRENCY_BOTTOM_SHEET) //TODO: Remove after CiCo currency bottom sheet is rolled out
          || isCicoToken(tokenInfo.symbol))
```
I think your logic is buggy here. Please add a regression test. 

Also, using a boolean expression to gate whether to return a boolean is not necessary-- you can just return the expression itself

Reply:
if the bottom sheet is not shown, that's when it matters that `isCicoToken(tokenInfo.symbol)` evaluates to true

Reply:
@cajubelt I thought it mattered in the case when the bottom sheet is shown? When the bottom sheet is not shown, this helper is not used since the token list is hard coded. I thought the idea was to have this here so that if the bottom sheet is used it still shows the same tokens as when the bottom sheet is not used, in the case that a new CICO token is added.

Reply:
that makes sense, I think I was a bit confused by Satish's comment because he said to retain the check when the feature flag is true, but looking closer it seems like the check is necessary on the token details screen only when the flag is false, so I changed it to be that case like you suggested.